src/neo/SmartContract/Native/Oracle/OraclePolicyContract.cs

```diff
@@ -42,7 +42,6 @@ public OraclePolicyContract()
         internal override bool Initialize(ApplicationEngine engine)
         {
             if (!base.Initialize(engine)) return false;
-            if (GetPerRequestFee(engine.Snapshot) != 0) return false;
```

Comment:
It follows the pattern of other Native Contract, check the storage for avoid duplicate initializations

Reply:
I added this check and the entire UT runs timed out